### DISCUSSION TITLE
After every start grafana connects to google & github

### DISCUSSION THREAD

**Discussion by kamzar1** [grafana/grafana#54682] [Alerting]
grafana-9.1.2-r0 on Alpine Linux

```
tcp        0      0 127.0.0.1:3000          0.0.0.0:*               LISTEN      1941/grafana-server
tcp        0      0 10.0.4.229:46136        34.120.177.193:443      ESTABLISHED 1941/grafana-server (google)
tcp        0      0 10.0.4.229:39750        185.199.111.133:443     ESTABLISHED 1941/grafana-server (github)
```
Is there any reason for that?

**Comment by kamzar1** [comment]
Solved:
/etc/grafana.ini
```
[analytics]
;reporting_enabled = true
reporting_enabled = false
```

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
